pkg/kubelet/dockertools/fake_docker_client.go

```diff
@@ -422,18 +422,14 @@ func (f *FakeDockerClient) Logs(opts docker.LogsOptions) error {
 
 // PullImage is a test-spy implementation of DockerInterface.PullImage.
 // It adds an entry "pull" to the internal method call record.
-func (f *FakeDockerClient) PullImage(opts docker.PullImageOptions, auth docker.AuthConfiguration) error {
+func (f *FakeDockerClient) PullImage(imageID string, auth dockertypes.AuthConfig, opts dockertypes.ImagePullOptions) error {
 	f.Lock()
 	defer f.Unlock()
 	f.called = append(f.called, "pull")
 	err := f.popError("pull")
 	if err == nil {
-		registry := opts.Registry
-		if len(registry) != 0 {
-			registry = registry + "/"
-		}
 		authJson, _ := json.Marshal(auth)
-		f.pulled = append(f.pulled, fmt.Sprintf("%s%s:%s using %s", registry, opts.Repository, opts.Tag, string(authJson)))
+		f.pulled = append(f.pulled, fmt.Sprintf("%s:%s using %s", imageID, opts.Tag, string(authJson)))
```

Comment:
I think it would be better to just use the `imageID`, because we'll bump up the engine-api version soon after this gets merged. In newer version of engine-api, there is no `imageID` in `PullImageOptions` https://github.com/kubernetes/kubernetes/pull/23800/files#r60776433